packages/pages/src/components/analytics/Analytics.ts

```diff
@@ -179,7 +180,7 @@ export class Analytics implements AnalyticsMethods {
   trackClick(
     eventName: string,
     conversionData?: ConversionDetails
-  ): (e: MouseEvent) => Promise<void> {
+  ): (e: MouseEvent<HTMLAnchorElement>) => Promise<void> {
```

Comment:
Should we also support button elements?

Reply:
so i was wondering about this, `buttons` should not navigate and trackClick won't actually work if you put it on a button as it needs the `href` to figure out what it's going to do. As it stands right now if there is now `href` it behaves just like `track()` so people should really only use this on an `a` element.

Reply:
so i guess i'm saying, No, a button should use `track` and not `trackClick`